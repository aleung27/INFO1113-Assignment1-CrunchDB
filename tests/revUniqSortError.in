SET a 4
REV A
UNIQ A
SORT A
POP a
REV a
UNIQ a
SORT a
BYE
