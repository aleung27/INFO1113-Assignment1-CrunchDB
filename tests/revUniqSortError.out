> ok

> no such key

> no such key

> no such key

> 4

> ok

> ok

> ok

> bye
